lib/chef/resource/windows_task.rb

```diff
@@ -94,22 +97,41 @@ def numeric_value_in_string?(val)
         false
       end
 
+      def validate_frequency_monthly(frequency_modifier, months, day)
```

Comment:
Can we avoid multiple if / if / unless here? We can write it in a single statement is it?

Reply:
fixed